mozilla/blurts-server
# Mntor 3250

# References:

Jira: MNTOR-3250
Figma: https://www.figma.com/design/YqpVpUbSTzZMj9Jbp2po8Q/Mozilla-Monitor-Premium-LP?node-id=1942-12535&t=DnOFixpfMw7vf82s-0

# Description
New sub-navbar for the landing page (US users only)

# Screenshot (if applicable)
<img width="1668" alt="Screenshot 2024-06-20 at 7 01 08 AM" src="https://github.com/mozilla/blurts-server/assets/31937894/0f665b83-0ec2-4c95-b7ef-ee5582c79fdb">

# How to test
Navigate to the landing page as a US user.  See the navbar just below the header.  Confirm that links work as expected.

# Checklist (Definition of Done)
